### DISCUSSION TITLE
Inline Images & Text Alignment

### DISCUSSION THREAD

**Discussion by amagrude** [gonzalezreal/swift-markdown-ui#317] [General]
Is there some way to top align inline images and have the text flow around them? 

I'm able to get to partway with what I want with tables, but not all the way there. Basically what I would like to do is to be able to drop images into various positions (left align, center, and right align) and have the text conditionally top or bottom align with them and wrap around them. Put another way, I have a rectangular region of text, let's say a block quote, that I want to be able to put an image into any corner and have the text flow around it.

I'm not seeing how to do this with inline images at all. I do see how to implement resizing which would allow me to lock the images at a fixed size, but not how to affect alignment. I'm at the edge of my SwiftUI understanding which compounds my confusion.

Can anyone give me a nudge in the right direction? If I need to find a non-Markdown-based solution, that's good know too...

Thanks!

**Comment by laike9m** [comment]
Similar question. I want to center an inline image but didn't find how.

**Reply by laike9m** [reply]
@gonzalezreal for some help

**Comment by rafaelpedretti-toast** [comment]
Yeah, with current latex responses from AI models it would be great to be able to vertically align the text

![image](https://github.com/user-attachments/assets/309daac8-63f8-4b2b-8c62-dd1f311aa2ea)
